src/lua/msgpack.c

```diff
@@ -510,7 +536,13 @@ lua_msgpack_encode(lua_State *L)
 	mpstream_init(&stream, buf, ibuf_reserve_cb, ibuf_alloc_cb,
 		      luamp_error, L);
```

Comment:
Without using an x alloc function or setting an error flag here, this patch doesn't seem to be complete - `luamp_error` may raise.

Reply:
This will be done in the scope of issue #8679